### PR TITLE
DX-3178, DX-3179, DX-3180 Refactored All BXML Verbs

### PR DESCRIPTION
**Pull request by brianluisgomez**

### CODE REVIEW

**Comment by brianluisgomez** [comment]
DX-3192, DX-3200, DX-3194

**Review by ajrice6713 — DISMISSED** [review]
Looks great!

File: pom.xml [hunk 1]
```diff
@@ -45,27 +45,31 @@
                         <arg>-Xlint:all</arg>
                         <arg>-J-Xss4m</arg><!-- Compiling the generated JSON.java file may require larger stack size. -->
                     </compilerArgs>
+                    <forceJavacCompilerUse>true</forceJavacCompilerUse>
```
Comment (bpateldx) [inline pom.xml:48]: Reformat the file

File: src/test/java/org/openapitools/client/model/unit/bxml/BridgeVerbTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+public class BridgeVerbTest {
+    /**
+    * Setting up Variables
+    */
+    Bridge bridge = new Bridge().builder()
+    .targetCallId("+19198675309")
+    .bridgeCompleteUrl("https://example.com")
+    .tag("test")
+    .build();
+    public JAXBContext jaxbContext;
```
Comment (bpateldx) [inline src/test/java/org/openapitools/client/model/unit/bxml/BridgeVerbTest.java:29]: The `jaxbContext` field does not need to be defined at this level, only usage is in single test below. No need for it to be `public` either. Suggest running linter like `SonarLint` to easily find such issues.

File: src/test/java/org/openapitools/client/model/unit/bxml/BxmlVerbTest.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    Bxml bxml = new Bxml().with(new Tag("Test"));
+    Tag tag = new Tag("Test Tag Content");
+    Bxml bxml2 = new Bxml().with(tag);
+
+    public JAXBContext jaxbContext;
+
+    @Test
+    public void toBxmlWorks() throws JAXBException {
+        jaxbContext = JAXBContext.newInstance(Bxml.class);
```
Comment (bpateldx) [inline src/test/java/org/openapitools/client/model/unit/bxml/BxmlVerbTest.java:38]: In this case, where `jaxbContext` is being referred in multiple Tests, you can initialize it at class level, no need to re-initialize unless you change configuration after constructing it.

**Pull request merged**